PySDM/formulae.py

```diff
@@ -51,6 +51,7 @@ def __init__(  # pylint: disable=too-many-locals
         isotope_diffusivity_ratios: str = "Null",
         isotope_relaxation_timescale: str = "Null",
         isotope_temperature_inference: str = "Null",
+        isotope_ventilation_ratio: str = "Brutsaert1982",
```

Comment:
suggest `Null` as a default value for al isotopic formulae, one of the reasons is to enforce that each example explicitly states the names of these parametrize upon Formulae instantiation